src/java.desktop/share/classes/javax/swing/JTree.java

```diff
@@ -4272,6 +4272,10 @@ protected class AccessibleJTree extends AccessibleJComponent
         TreePath   leadSelectionPath;
         Accessible leadSelectionAccessible;
 
+        static {
+            SwingAccessor.setAccessibleJTreeNodeCreateAccessor(new AccessibleTreNodeACreateccessor());
```

Comment:
I think there is a typo here. AccessibleTreNodeACreateccessor does not sound right. Should be AccessibleTreeNodeCreateAccessor.